app/app.go

```diff
@@ -155,6 +208,8 @@ var (
 
 		// Custom modules
 		profiles.AppModuleBasic{},
+
+		wasm.AppModuleBasic{},
```

Comment:
Since it's not a custom module, I think it would be better to put this after the `vesting` module